test/smoke/src/sql/main.ts

```diff
@@ -35,7 +35,7 @@ const PLATFORM = '${PLATFORM}';
 const RUNTIME = '${RUNTIME}';
 const VERSION = '${VERSION}';
 
-const sqliteUrl = `https://github.com/Microsoft/azuredatastudio-sqlite/releases/download/1.5.0/azuredatastudio-sqlite-${PLATFORM}-${RUNTIME}-${VERSION}.zip`;
+const sqliteUrl = `https://github.com/Microsoft/azuredatastudio-sqlite/releases/download/1.6.0/azuredatastudio-sqlite-${PLATFORM}-${RUNTIME}-${VERSION}.zip`;
```

Comment:
Updating this requires a distro change which me or Karl can do. 

But if web is using a different version then you'll need to modify this statement to pull down that version specifically for web, otherwise you'll be breaking the non-web tests (since they're using the current older verison of node)

@kburtram Do you know why web is using this different version of node in the first place?